MIGRATION.md

```diff
@@ -1,6 +1,21 @@
 # Migration Instructions for Major Releases
 
-### From `7.0.0` to `8.0.0`
+## Migrating to Trunk
+
+- Ensure that calls to `ios_bump_version_release` already passed `skip_glotpress: true`.
```

Comment:
From what I see in the CHANGELOG, this one landed in `trunk` via #443 already — and that's also probably why you removed it from line 9 of the CHANGELOG.md because it was already listed on line 80 under `7.0.0` changes, right?)

Which is confirmed if I look at the merge commit on GitHub to see which tags include that commit:
<img width="640" alt="image" src="https://github.com/wordpress-mobile/release-toolkit/assets/216089/7b6a918c-6922-47db-92e0-178750988909">

So that means that technically this should be part of the Migration from 6.x to 7.0, not from 8.0 to Trunk

Reply:
Ah, exactly, I updated the CHANGELOG but didn't realise this was also referring to it. Will change it, thanks for noticing!